src/navigations/Tabs.js

```diff
@@ -1,15 +1,38 @@
 import React from "react";
 import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
-import Home from "../screens/Home";
-import Category from "../screens/Category";
-import Feed from "../screens/Feed";
-import MyScreen from "../screens/MyScreen";
+import { Ionicons } from "@expo/vector-icons";
+import Home from "../../src/screens/Home/Home";
+import Category from "../screens/Category/Category";
+import Feed from "../screens/Feed/Feed";
+import MyScreen from "../screens/MyScreen/MyScreen";
 
 const Tabs = createBottomTabNavigator();
 
 export default () => {
   return (
-    <Tabs.Navigator>
+    <Tabs.Navigator
+      screenOptions={(route) => ({
```

Comment:
이렇게 비구조화 하면 아래에서 ```route.route``` 안 할 수 있겠네요!
```suggestion
      screenOptions={({route}) => ({
```

Reply:
수정했습니다!